catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -498,6 +499,15 @@ public static int getBit(int number, int index) {
 		return 0;
 	}
 
+	public static boolean isGlobalVariable(UserData<?> item) {
+		Project currentProject = ProjectManager.getInstance().getCurrentProject();
+		if (currentProject.getUserVariables().contains(item) || currentProject.getUserLists().contains(item)
+				|| currentProject.getMultiplayerVariables().contains(item)) {
+			return true;
```

Comment:
Please just return the condition here: *return currentProject.getUserVariables().contains(item) || ...*
And I would not place this method in the Utils class but rather in the Project class as an instance method.